sources/core/Stride.Core.AssemblyProcessor/Stride.Core.AssemblyProcessor.csproj

```diff
@@ -27,7 +27,7 @@
   
   <!-- Add support for building this project from the Project Directory -->
   <PropertyGroup>
-    <SolutionDir Condition="'$(SolutionDir)' == ''">$(ProjectDir)..\..\..\build\</SolutionDir>
+    <DependencyDir>$(ProjectDir)..\..\..\..\deps</DependencyDir>
```

Comment:
Just a little nitpick:
Shouldn't it be `DependenciesDir` instead? It's the directory where **all** Stride dependencies are located.

Reply:
Hi @Ethereal77, grammatically speaking you're not wrong, and if people have a strong preference it's an easy change to make.

That being said, historically the preference has been for readability/laziness rather than grammatical accuracy. ;-) As examples, there used to be the "src" and "source", folders. Still remaining are the "bin", and "obj" folders. In our own project hierarchy there's the "build" folder, though technically it's the "builds" folder. Those are all cases where the folder name acted as a sort/filter rather than as a description of the conents, ie. "this is where a source file goes," or "this is where a build file goes."

I also just found DependecyDir easier to read and say than DependenciesDir. But again, that's likely just a dialectical preference or laziness. If people prefer the latter, I'm happy to change it. Thoughts?

Reply:
Not a strong preference. I'm not a native english speaker, so it doesn't matter much to me. Anyway I agree with you on the simplicity point.
**Readability > all else** 😁